tests/io/test_postgis.py

```diff
@@ -291,6 +291,28 @@ def test_non_standard_column_names(self, example_positionfixes, conn_postgis):
         finally:
             del_table(conn, table)
 
+    def test_daylight_saving_tz(self, example_positionfixes, conn_postgis):
+        """Test if function can handle different tz informations in one column.
+
+        PostgreSQL saves all its datetimes in UTC and then on exports them to the local timezone.
+        That all works fine except when the local timezone changed in the past for example with daylight saving.
+        """
+        pfs = example_positionfixes.copy()
+        conn_string, conn = conn_postgis
+        table = "positionfixes"
+        sql = f"SELECT * FROM {table}"
+        t1 = pd.Timestamp("2021-08-01 16:00:00", tz="utc")  # summer time
```

Comment:
Did you check if they actually appear in the database as timestamps with different timezones? Timezone UTC should be without summer/winter time, shouldn't it?

Reply:
Yes, they don't appear in the database as timestamp with different timezones as everything is saved in UTC in postgres.
But I checked the export and there it created two UTC offsets one for the summer time and one for the winter time.

Reply:
Great, thanks!